python_bindings/setup.py

```diff
@@ -79,18 +79,19 @@ class BuildExt(build_ext):
     """A custom build extension for adding compiler-specific options."""
     c_opts = {
         'msvc': ['/EHsc', '/openmp', '/O2'],
-        'unix': ['-O3'],
+        'unix': ['-O3', '-msse4.2'],
```

Comment:
Ok. I also have a question regarding Windows build. It builds using cmake, right?